Makefile

```diff
@@ -137,15 +137,15 @@ configure-mini: amigaos.cmake link.sh CMakeLists.txt Dummy/libdummy.a ffmpeg/.bu
 		$(realpath ./))
 
 build:
-	(cd cross-build && make -j1)
+	(cd cross-build && make -j$(shell nproc))
```

Comment:
This might cause a problem for users that do not have "nproc". I ran into this in another repository that was using "nproc", but their machine did not have it.

Since this is already a recipe, you can just use "-j$$(nproc)" - just fyi.

But, you could also just use "-j" - but that provides no limit on parallel jobs.

Reply:
@3246251196 Made this change in my second commit